apps/web/app/connect/portal/page.tsx

```diff
@@ -56,25 +53,14 @@ export default async function PortalPage({
       <div>Authenticated user only. Your role is {getViewerId(viewer)}</div>
     )
   }
-
-  const headersList = headers()
-  const referer = headersList.get('referer')
-
-  const shouldRenderAutographPortal =
-    viewer.userId === 'xxx' || // TODO for future?
-    referer?.includes('withautograph.com') ||
-    referer?.includes('https://current-owl-78.accounts.dev') ||
-    referer?.includes('ag-frontend-staging.onrender.com') ||
-    searchParams['autograph'] === 'true' // not sure if this will work
+  const shouldRenderAG =
+    viewer.orgId === 'org_2lcCCimyICKI8cpPNQt195h5zrP' ||
```

Comment:
I think this should be a param rather than hardcoded. We accept a whole bunch of url params, chief among which is `token`, and can simply expand upon it.

Reply:
i don't think we'd want customer A to load the customer B UI by passing a param... at least this way there needs to be some auth?

Reply:
I think there are different UI variants, but no customer A ui vs. customer B ui. Also while the UI style is different, I don't see where auth comes in. Each user can still only see their own data.